packages/cypress/src/support.ts

```diff
@@ -1,5 +1,5 @@
 import type { TestMetadataV2 } from "@replayio/test-utils";
-import { buildTestId } from "@replayio/test-utils";
+import { buildTestId } from "@replayio/test-utils/testId";
```

Comment:
Allowing this is the core of the fix. 

Currently `@replayio/test-utils` depends on `@replayio/replay` so when we were importing from the root entrypoint we were also indirectly importing the whole `@replayio/replay`. That created problems for Cypress' webpack bundling